neurons/validator.py

```diff
@@ -172,10 +174,7 @@ async def _update_and_get_miner_index(
 
     def _get_miner_index(self, hotkey: str) -> Optional[ScorableMinerIndex]:
```

Comment:
Is it worth removing this helper method now?

Reply:
Removed. Good catch.